src/docfx/lib/PathUtility.cs

```diff
@@ -205,6 +205,24 @@ public static string UrlToShortName(string url)
             return result.ToString();
         }
 
+        public static string ChangeExtension(string filePath, string extension, string[] acceptableExtension = null)
+        {
+            acceptableExtension = acceptableExtension ?? new string[] { "raw.page.json", "mta.json" };
```

Comment:
`raw.page.json` and `mta.json` this is for legacy only (and will be removed shortly once docs-publish) is done, so I suggest to not extract this shared method and don't change code for non-legacy files for this fix

Reply:
so where should we put the shared method used in legacy code

---
In reply to: [284538603](https://github.com/dotnet/docfx/pull/4560#discussion_r284538603) [](ancestors = 284538603)